worker/internal/benthos/transformers/transform_character_scramble_test.go

```diff
@@ -2,6 +2,7 @@ package transformers
 
 import (
```

Comment:
Love the regex tests in this file. Can we have a test specifically for emails?

Something like:
```go
^(\w+)@(gmail\.com|yahoo\.com|nucleus\.com)$
```
<img width="431" alt="image" src="https://github.com/nucleuscloud/neosync/assets/2420177/ceb8bfd7-644f-4886-b6a3-ed86e08f68bb">

Reply:
updated! had to change your regex a little bit since that matches the entire email string and just returns random text then, so updated it just check for the email domain